persistence-cassandra/src/main/scala/com/evolutiongaming/kafka/flow/snapshot/CassandraSnapshots.scala

```diff
@@ -56,7 +56,12 @@ class CassandraSnapshots[F[_]: MonadThrowable: Clock, T](
       _ <- session.execute(boundStatement).first.void
     } yield ()
 
-  def get(key: KafkaKey): F[Option[KafkaSnapshot[T]]] = {
+  def get(key: KafkaKey): F[Option[KafkaSnapshot[T]]] =
+    getLegacy(key) flatMap { snapshot =>
```

Comment:
it should be the other way round, getNew and fall back to getLegacy, or am I missing something here?

Reply:
Yes, I fixed it :)